azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -625,6 +632,41 @@ public synchronized void prefetchToken(final File tokenFile,
       cred.addToken(jhsdt.getService(), jhsdt);
     }
 
+    if(props.getBoolean(OBTAIN_HIVESERVER2_TOKEN)){
+      Connection conn = null;
+      Token<DelegationTokenIdentifier> hive2Token = null;
+      try {
+        Class.forName("org.apache.hive.jdbc.HiveDriver");
```

Comment:
I don't think this line is necessary for modern drivers. See documentation here: https://docs.oracle.com/javase/6/docs/api/java/sql/DriverManager.html

@kunkun-tang what do you think?